### ISSUE TITLE
Bug while working with units and unit scale

### ISSUE THREAD

**Issue by jantma2** [alessandro-zomparelli/tissue#16]
Hi,

thank you for the great work you did with that tissue add-on.
Currently I'm running into this issue.
I'm working on data, that came from CAD models. In order to have them in correct dimensions (necessary for physics based renderings - in my case optics) I've to use Blender's unit capabilities. Usually they are set to pre-defined "Millimeters".

In that case there seems to be some trouble with the tessaltion option in this add-on.
Results look totally destroyed. 
Please have look into that. Thanks a lot.

**Comment by alessandro-zomparelli** [comment]
I'm closing it because in Blender 2.80 units are natively used. I'll reopen it if other issues will occur.

**Comment by jantma2** [comment]
Hi Alessandro,

I've spent some hours during the last days to figure out what I may have
ment with that post... I failed. :-)

At the moment I'm using your add on and it's working perfectly fine. Thanks
for your efforts.

You're right with closing that issue.
I wish some nice days between years and a Happy New Year.

With best regards
Martin

Am Do., 27. Dez. 2018, 13:09 hat Alessandro Zomparelli <
notifications@github.com> geschrieben:
